### PR TITLE
#167165010 Set Up React Base Project

### PR DESCRIPTION
**Pull request by Tunji17**
### What does this PR do?
Sets up the base template for our project

#### Description of Task to be completed?
adds webpack 
adds babel
adds eslint

#### How should this be manually tested?

> git clone repo 
> run npm install
> run npm run dev

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167165010](https://www.pivotaltracker.com/story/show/167165010)

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "react-hot-loader": "4.6.3",
+    "react-test-renderer": "16.7.0",
+    "style-loader": "0.23.1",
+    "uglifyjs-webpack-plugin": "2.1.1",
+    "webpack": "4.28.4",
+    "webpack-cli": "3.2.1",
+    "webpack-dev-server": "3.1.14",
+    "webpack-merge": "4.2.1"
+  }
+}
```
Comment (mazma1) [inline package.json:55]: All these plenty dependencies.. 👀

Comment (mazma1) [reply package.json:55]: @Tunji17 please squash your commits

Comment (Tunji17) [reply package.json:55]: I have done that, I was having issues with my internet and had to go to IT @mazma1

Comment (mazma1) [reply package.json:55]: Yes @Tunji17, you have 1 commit now. But still, peep the commit message and _refactor_ 
Screenshot 2019-08-26 at 6 43 35 PM

**Review by tolulope-od — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Tunji17) [reply package.json:55]: @mazma1  I have refactored the commit message

**Pull request merged**